### ISSUE TITLE
Heroku deployment response time alert

### ISSUE THREAD

**Issue by victorlin** [nextstrain/nextstrain.org#1064]
Over the past few days we have been getting several emails ([e.g.](https://bedfordlab.slack.com/files/USLACKBOT/F07V7HGJN5D/_alert__nextstrain-server__response_times__web_)) with the following message:

> The 95th percentile response time for [nextstrain-server](https://dashboard.heroku.com/apps/23ea4905-f8ed-4c13-889f-9f8f8c8f4cd1/metrics/web?starting=2-hours-ago) has exceeded your threshold setting of 1000ms.
> 
> Historical response time data is available in [Application Metrics](https://dashboard.heroku.com/apps/23ea4905-f8ed-4c13-889f-9f8f8c8f4cd1/metrics/web?starting=2-hours-ago). Additional guidance on correcting slow requests and request timeouts is available in [Dev Center](https://devcenter.heroku.com/articles/request-timeout).

They have all self-resolved within minutes, but still out of the ordinary.

Here is a screenshot of metrics over the past week. I've turned off max response time to make the percentiles more readable.

<img width="837" alt="image" src="https://github.com/user-attachments/assets/beac356e-46ce-4f2c-8c59-782c3ad248c9">

## Tasks

1. Investigate difference in behavior
2. If this is expected, increase the alert threshold.
    Otherwise, take action to reduce response times.

**Comment by victorlin** [comment]
Another wave of alerts came in today: [first message](https://bedfordlab.slack.com/archives/C016UMRR02W/p1731443250861369), metrics screenshot

<img width="948" alt="image" src="https://github.com/user-attachments/assets/df05b76e-3eda-4285-87cb-6115153ef8fd">

**Comment by victorlin** [comment]
If these are all Auspice pages, it could be explained by https://github.com/nextstrain/auspice/issues/1919

**Comment by tsibley** [comment]
@victorlin I don't think that makes sense. The response times Heroku is measuring are the duration of individual HTTP request/response pairs between their routing/reverse proxy layer and our app on their dynos. It's not measuring "page load time" in the browser.

**Comment by victorlin** [comment]
@tsibley oh right, thanks for clarifying.
